Health/src/main/java/com/sciencesquad/health/health/Nutrition/RealmNutritionModel.java

```diff
@@ -0,0 +1,57 @@
+package com.sciencesquad.health.health.Nutrition;
+
+
+import android.support.annotation.NonNull;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+
+/**
+ * Created by danielmiller on 2/11/16.
+ *
+ * Realm Model for Nutrition database.
+ * Things that are relevant to store for the Nutrition Module go here.
+ */
+public class RealmNutritionModel extends RealmObject {
+
+    private static final String TAG = "Realm Nutrition Model";
+
+    /**
+     * Nutrition Model data summary.
+     * - calorieIntake: calories taken in on a certain day.
+     * - date: Calendar date where this model was created.
+     *  Note: Particularly useful when compiling data in history.
+     */
+
+    // we can set up the primary key to something else if need be.
+    @PrimaryKey
+    private int calorieIntake;
+
+    // other relevant data
+    @NonNull
+    private Date date;
+
+    public RealmNutritionModel(){
+        this.calorieIntake = 0;
+        Calendar rightNow = Calendar.getInstance();
+        this.date = rightNow.getTime();
+    }
+
+    public int getCalorieIntake(){ return calorieIntake; }
```

Comment:
For any Model class, only `get...` methods are desired, since you never want to be modifying data without realizing it. Also, use `@Value.Immutable` which generates everything for you.